### PR TITLE
Improvement/s3 c 3971 reduce event footprint

### PR DESCRIPTION
**Pull request by tmacro**
package.json will be changed after the related Utapi PR has merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "node-uuid": "^1.4.3",
     "npm-run-all": "~4.1.5",
     "sproxydclient": "scality/sproxydclient#44f025b",
-    "utapi": "scality/utapi#eef4623",
+    "utapi": "scality/utapi#improvement/S3C-3971_reduce_event_footprint",
```
Comment (naren-scality) [inline package.json:37]: is this temporary?

**Review by naren-rajendran — APPROVED** [review]

**Review by miniscruff — CHANGES_REQUESTED** [review]
Marking as needs work for package.json update, but looks good otherwise.

**Comment by tmacro** [comment]
@miniscruff package.json has been updated

**Review by miniscruff — APPROVED** [review]

**Pull request merged**